fluent-syntax/test/fixtures_structure/unclosed_empty_placeable_error.json

```diff
@@ -0,0 +1,112 @@
+{
+  "type": "Resource",
+  "body": [
+    {
+      "type": "Comment",
+      "content": "~ERROR E0003, pos 8, args \"}\"",
```

Comment:
I'd like to leave it as a separate file. It tests that the error position of the following:

```properties
foo = {
bar = Bar
```

….is at the first letter of `bar` rather than at the `=` after it. When parsing `foo`, `bar` correctly parses as a message reference. The parser has special-purpose logic to back-track to the beginning of the line to gracefully recover from the syntax error and not lose the valid `bar` message definition.